.cspell/en-words.txt

```diff
@@ -47,6 +47,7 @@ frontendproxy
 github
 gitpod
 grafana
+Greptime
```

Comment:
As mentioned in another comment, please revert the changes to this file. Thanks!